src/errors.js

```diff
@@ -31,6 +31,12 @@ export class RemoteTempInstallNotSupported extends WebExtError {
   }
 }
 
+export class UsageError extends WebExtError {
```

Comment:
Also, since `InvalidManifest` is something the developer can fix, you should make this a `UsageError` by changing its base class like:
````javascript
export class InvalidManifest extends UsageError {
}
````